src/app/core/store/user/user.reducer.ts

```diff
@@ -15,7 +15,7 @@ import {
 } from './user.actions';
 
 const initialUserState: UserState = {
-  currentAccess: 'guest',
+  currentAccess: 'manager',
```

Comment:
This line must have been left over from testing and needs to be changed back.

Reply:
Yeah, I forget about this 😬